test/integration/ProjectBrowserTest.kt

```diff
@@ -34,13 +37,13 @@ class ProjectBrowserTest: IntegrationTest() {
         studioTest(composeRule) {
             val createdDirectoryName = "created"
 
-            cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
+            cloneAndOpenProject(composeRule, source = Utils.SAMPLE_DATA_PATH, destination = testID)
```

Comment:
Import `SAMPLE_DATA_PATH` directly from `Utils` to get back to
```suggestion
            cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
```
Same for the other uses of `Utils` in this PR

Reply:
Done.